src/runtime/schema.ts

```diff
@@ -19,10 +19,21 @@ import {Referenceable} from './crdt/crdt-collection.js';
 import {Singleton} from './handle.js';
 import {CRDTSingleton} from './crdt/crdt-singleton.js';
 
+export interface Field {
+  // tslint:disable-next-line:no-any
+  types?: any[];
+  // tslint:disable-next-line:no-any
+  model?: any;
+  // tslint:disable-next-line:no-any
+  location?: any;
```

Comment:
There seem to be a bunch of things here that come directly from the parser and should not be present in the type of a schema.

All you need I believe is a `type` and `kind`. Do you need anything more?